java/code/src/com/suse/manager/webui/templates/minion/recurring-actions.jade

```diff
@@ -10,6 +10,7 @@ script(type='text/javascript').
     window.csrfToken = "#{csrf_token}";
     window.minions = [{id: #{server.id}, name: "#{server.name}"}];
     window.entityType = "#{entityType}";
+    window.isControlNode = "#{isControlNode}";
```

Comment:
I don't recall the syntax off the top of my head, but I think this can be a regular boolean value, not a string like this:

```suggestion
    window.isControlNode = #{isControlNode};
```

Then we can check against `true` elsewhere in the code, which feels a bit more natural imo.